darkmoon/api/v1/metadata/views.py

```diff
@@ -60,20 +67,14 @@ async def list_metadata(
             hash_parameter = "hashes." + str(hash_type)
             search[hash_parameter] = hash
         elif hash_type:
-            raise HTTPException(status_code=400, detail="Enter hash")
+            raise IncorrectInputException(status_code=422, detail="Incorrect input.")
```

Comment:
You can be more specific with your messages, remember the better your message the faster you can debug what's going on. The old messages were OK

Reply:
All the messages could use improvement.